server/src/creators.js

```diff
@@ -106,10 +106,14 @@ export const getCreators = (models) => {
       }
       return Group.create({
         name,
+        icon: icon || 'group',
         organisationId: organisation.id,
-      }).then(
-        group => group.addUsers(users)
-          .then(() => group));
+      }).then(group => Promise.all([
```

Comment:
@sdunster this seems to be returning the group before the tags have been added, as in the UI will show now tags on the group until you refetch the data. any ideas?